day_01.py

```diff
@@ -5,35 +5,23 @@ def read_file():
 
 def do_part_one():
     lines = read_file()
-    num_increases = 0
-
-    for idx, current_measurement in enumerate(lines):
-        previous_measurement = lines[idx - 1]
-        if current_measurement > previous_measurement:
-            num_increases += 1
+    num_increases = sum(a < b for a, b in zip(lines, lines[1:]))
```

Comment:
`zip(lines, lines[1:])` is a way to compare an element, `a`, with the element that follows it `b`. 

`lines` and `lines[1:]` are of different lengths, but `zip` will stop when the shorter/shortest iterable is exhausted, so it just stops after `a` is the penultimate element and `b` is the last element of the list. 

The `sum()` builtin when given an iterable of boolean values will count the number of `True` elements, so this will count the number of times `a` is less than `b`.